app/controllers/api/v1/students_controller.rb

```diff
@@ -12,11 +12,16 @@ def show
       end
 
       def create
-        Student.create(student_params)
-        render json: StudentSerializer.new(Student.last)
+        # Student.create(student_params)
+        # render json: StudentSerializer.new(Student.last)
```

Comment:
Are we saving this for later or can it be removed?

Reply:
Uh oh! This can be removed